### PR TITLE
Add unit test for SPDM 1.3 DIGEST: KeyPairID and CertificateInfo

### PR DESCRIPTION
**Pull request by Xiaohanjlll**

### CODE REVIEW

File: unit_test/test_spdm_requester/get_digests.c [hunk 1]
```diff
@@ -1577,6 +1760,204 @@ static void libspdm_test_requester_get_digests_case26(void **state)
...
+    m_local_cert_info[0] = SPDM_CERTIFICATE_INFO_CERT_MODEL_DEVICE_CERT;
+    m_local_key_usage_bit_mask[0] = 0;
+
+    libspdm_zero_mem(total_digest_buffer, sizeof(total_digest_buffer));
+    status = libspdm_get_digest(spdm_context, NULL, &slot_mask, &total_digest_buffer);
+    assert_int_equal(status, LIBSPDM_STATUS_INVALID_MSG_FIELD);
+
+    /* Sub Case 2: Set CertificateInfo to 0*/
+    m_local_key_pair_id[0] = 0x00;
+    m_local_cert_info[0] = 0;
```
Comment (jyao1) [inline unit_test/test_spdm_requester/get_digests.c:1904]: 0 is valid. If you want to trigger invalid msg, please use GENERIC_CERT.

Comment (Xiaohanjlll) [reply unit_test/test_spdm_requester/get_digests.c]: Thanks, I have fix the problem

File: unit_test/test_spdm_requester/get_digests.c [hunk 2]
```diff
@@ -1577,6 +1759,204 @@ static void libspdm_test_requester_get_digests_case26(void **state)
...
+    m_local_key_pair_id[2] = 0x02;
+    m_local_cert_info[2] = SPDM_CERTIFICATE_INFO_CERT_MODEL_GENERIC_CERT;
+    m_local_key_usage_bit_mask[2] = SPDM_KEY_USAGE_BIT_MASK_MEASUREMENT_USE;
+
+    m_local_key_pair_id[3] = 0x03;
+    m_local_cert_info[3] = SPDM_CERTIFICATE_INFO_CERT_MODEL_DEVICE_CERT;
+    m_local_key_usage_bit_mask[3] = SPDM_KEY_USAGE_BIT_MASK_ENDPOINT_INFO_USE;
+
+    m_local_key_pair_id[4] = 0x04;
+    m_local_cert_info[4] = SPDM_CERTIFICATE_INFO_CERT_MODEL_NONE;
```
Comment (jyao1) [inline unit_test/test_spdm_requester/get_digests.c:1815]: case bug. Digest does not set to all 0.

File: unit_test/test_spdm_responder/encap_get_digests.c [hunk 3]
```diff
@@ -320,6 +320,136 @@ void test_spdm_responder_encap_get_digests_case6(void **state)
...
+    key_pair_id[2] = 0x02;
+    cert_info[2] = SPDM_CERTIFICATE_INFO_CERT_MODEL_GENERIC_CERT;
+    key_usage_bit_mask[2] = SPDM_KEY_USAGE_BIT_MASK_MEASUREMENT_USE;
+
+    key_pair_id[3] = 0x03;
+    cert_info[3] = SPDM_CERTIFICATE_INFO_CERT_MODEL_DEVICE_CERT;
+    key_usage_bit_mask[3] = SPDM_KEY_USAGE_BIT_MASK_ENDPOINT_INFO_USE;
+
+    key_pair_id[4] = 0x04;
+    cert_info[4] = SPDM_CERTIFICATE_INFO_CERT_MODEL_NONE;
```
Comment (jyao1) [inline unit_test/test_spdm_responder/encap_get_digests.c:411]: Case bug

File: unit_test/test_spdm_requester/get_digests.c (hunk 2 above)
Comment (Xiaohanjlll) [reply unit_test/test_spdm_requester/get_digests.c]: Thanks, I have fix the bug

File: unit_test/test_spdm_responder/encap_get_digests.c (hunk 3 above)
Comment (Xiaohanjlll) [reply unit_test/test_spdm_responder/encap_get_digests.c]: Thanks, I have fix the bug

**Pull request merged**